model/otel/registry.yaml

```
@@ -39,6 +39,23 @@ groups:
               stability: development
         brief: "The result value of the sampler for this span"
         stability: development
+      - id: otel.span.parent.origin
+        type:
+          members:
+            - id: none
+              value: NONE
```

does it have to be in SCREAMING_CASE?


```suggestion
              value: none
```

also is it necessary to set this attribute for root spans at all?